### DISCUSSION TITLE
Is this FCS152 flashable with ESP32S2 and Rev 2.1.1226

### DISCUSSION THREAD

**Discussion by CerberusSolutions** [tarxvftech/FCS-152#71] [General]
Hello all, love the FCS152, nice operation/bright display and tbh not bad performance, stumbled across this on here and wondered if this  has a bin file that can be loaded with the radio? And if there is a change log at all? Happy to test on the hardware if I can find a way to load it.

Would love wider coverage from the radio - I am sure the module is rated way beyond the RF bands it comes with.

**Comment by tarxvftech** [comment]
No, the only release here is a fix for the frequency entry bug. OEM firmware fixed that shortly after, too - my fix was just to remove the at-input-time input filtering. I believe that's documented in the README. The rest was a start on some rewrites but I lost interest.


Do you have any evidence for the bit about the RF module? 
What does it say if you send it a AT+DMOSETGROUP for those other bands?
